README.md

```diff
@@ -1 +1,69 @@
 # Noa Assistant
+ This repository conatains all AI stuff of Noa that can be used in parallel with\
+ a Authentication server to protect APIs and throttled usage\
+Working features:
+1. Conversational AI
+2. Coversation with Photos
+3. Input from Audio
+4. Websearch
+
+### Setup
+1. copy the `.env.example` file to `.env` and fill in the keys
+Note: for reverse image search a image cdn needed, you can use any api which accepts image and returns a url.\
+format of the api should be like this.\
+Or this can be avoided if not using reverse image search
+```sh
+curl -F'file=@path/to/yourfile.png' -Fexpires=2 example.com
+```
+
+1. Create and activate a python virtual enviroment(optional). eg. from [freecodecamp](https://www.freecodecamp.org/news/how-to-setup-virtual-environments-in-python/)
+
+2. Install [ffmpeg](https://ffmpeg.org/download.html), and make sure its available in PATH variable
```

Comment:
Does noa-assistant really need this? I thought we transmit the raw bytes without any decoding on the server.

Reply:
Yup for whisper its an overhead, it allows any file format btw, if we fix one format then it will be not needed